terraformInstall.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+## CONDITIONAL 1
+echo "Installing prerequisites..."
+sudo apt-get update && sudo apt-get install -y gnupg software-properties-common
+
+#!/bin/bash
+
+# Check if gpg is installed
+if ( which gpg > /dev/null 2>&1 )
```

Comment:
for gpg you might want to use `dpgk -s gnupg` since that's the one you're installing.